client/src/partials/journal/journal.html

```diff
@@ -2,7 +2,21 @@
   <div class="bhima-title">
     <ol class="headercrumb">
       <li class="static">{{ "TREE.FINANCE" | translate }}</li>
-      <li class="title">{{ "POSTING_JOURNAL.TITLE" | translate }}</li>
+      <li class="title">
+        {{ "POSTING_JOURNAL.TITLE" | translate }}
+
+        <span class="badge badge-primary text-uppercase" ng-show="JournalCtrl.transactions.isEditing()">
+          <i class="fa fa-street-view"></i>
+          Edit Mode
+        </span>
+
+        <span class="badge badge-warning text-uppercase" ng-show="JournalCtrl.transactions._highlights.length > 0">
+          {{ JournalCtrl.transactions._highlights.length }}
+          <span ng-if="JournalCtrl.transactions._highlights.length == 1">Highlight</span>
```

Comment:
They were mostly for prototyping, although they may be useful in highlighting transactions is coming from a previous state.  Something like:

`#/journal/:uuid` --> highlight transaction with record UUID `:uuid`.

But I'll take this out for the time being.  They are easy to add back in.